reference/config_files/system_tools_warning.rst

```diff
@@ -0,0 +1,4 @@
+:orphan:
+
+.. warning::
```

Comment:
I think the idea is to have a generic warning saying "This feature is experimental and subject to breaking changes." and reuse it in other parts of the docs

Reply:
yes, move it to first level in the docs, so it can be reused anywhere.

I have added a few experimental warnings, that were specific about specific parts (like json output) so I couldn't reuse a general one. But others should be reused.